src/lib/functions/utils.js

```diff
@@ -21,8 +19,40 @@ const detectAwsSdkError = ({ error }) => {
 
 const formatLambdaError = (err) => chalk.red(`${err.errorType}: ${err.errorMessage}`)
 
+const exceptionsList = new Set([
```

Comment:
I'm assuming this list comes from production code? If so, do you mind adding a comment to that effect?, just so people know where to look if they're working on this?

Reply:
Sure thing! It stems from https://github.com/netlify/proxy/blob/main/pkg/functions/request.go#L105, which I put into one of the commit's bodies - but a comment is a good idea, too.

```suggestion
// should be equivalent to https://github.com/netlify/proxy/blob/main/pkg/functions/request.go#L105
const exceptionsList = new Set([
```